LISA/Executor.cpp

```diff
@@ -186,16 +186,16 @@ void Executor::taskRunner(std::function<void()> task)
     INFO(task, " started ");
     try {
         task();
-        // TODO send status "Done"
         INFO(task, " done");
+        operationStatusCallback(currentTask.handle, OperationStatus::SUCCESS, std::string{});
```

Comment:
currentTask should be accessed under taskMutex - it might used from different threads

Reply:
Good catch, thanks. Fixed